lib/clusterManager.js

```diff
@@ -120,6 +120,10 @@ if (cluster.isMaster) {
     var inits = [];
     inits.push(this.initSecret.bind(this));
     inits.push(this.initSecret.bind(this));
+    if (process.env.ZOWE_SESSION_SECRET) {
+      this.secret = process.env.ZOWE_SESSION_SECRET;
+      inits.pop();
```

Comment:
that pops loopbackSecret, not secret.
the initSecret function is weird.
loopbackSecret is a way to make it so that some dataservices can only be reached internally instead of by a browser.

Reply:
I checked that it works. I need a single call to `initSecret()` and the call initializes `loopbackSecret` because `secret` is already initialized.